### PR TITLE
[LPD-22337 | PDI] Create a playwright test

### PR DESCRIPTION
**Pull request by davisaints**
- **JIRA Ticket:** [LPD-22337](https://liferay.atlassian.net/browse/LPD-22337)

This is a regression test designed to verify that the following bug has been successfully fixed: [LPS-175992](https://liferay.atlassian.net/browse/LPS-175992)

- Test Report:

Playwright-Test-Report

### CODE REVIEW

File: modules/test/playwright/helpers/ObjectAdminApiHelper.ts [hunk 1]
```diff
@@ -119,6 +119,40 @@ export class ObjectAdminApiHelper {
 		);
 	}
 
+	async postFormulaObjectField(
```
Comment (brianwulbern) [inline modules/test/playwright/helpers/ObjectAdminApiHelper.ts]: @davisaints does this need to be alphabetized?  I see other function names tend to follow that pattern, but `postIntegerObjectField` is also out of place

**Comment by brianwulbern** [comment]
@davisaints this looks good overall!  (I also don't 100% understand all the details, but I can follow the purpose of the test with your added logic.)

Do you have a test report / terminator output showing the test ran successfully?

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

File: modules/test/playwright/helpers/ObjectAdminApiHelper.ts (hunk 1 above)
Comment (davisaints) [reply modules/test/playwright/helpers/ObjectAdminApiHelper.ts]: @brianwulbern, yes, it must be in alphabetical order. I had considered this and thought that the source format would handle it, as it does in the `.testcase` and `.macro` files. I did it in [e7cd9](https://github.com/brianwulbern/liferay-portal/pull/437/commits/e7cd94f23e84083468fef76453e0641500939973).

**Comment by davisaints** [comment]
> @davisaints this looks good overall! (I also don't 100% understand all the details, but I can follow the purpose of the test with your added logic.)
> 
> Do you have a test report / terminator output showing the test ran successfully?

@brianwulbern the pull request description contains the test report, confirming successful execution.

**Comment by brianwulbern** [comment]
LGTM!  Nice work, @davisaints

**Comment by brianwulbern** [comment]
ci:forward

**Comment by davisaints** [comment]
ci:stop

**Pull request closed without merging**